routers/dashboard/analytics.py

```
@@ -0,0 +1,1475 @@
+            else:
+                neutral, positive, negative, interactions = 0, 0, 0, 0
+
+                for message in current_records:
+                    try:
+                        if message['agent_sentiment'] == 'Neutral':
+                            neutral += 1
+                        elif message['agent_sentiment'] == 'Positive':
+                            positive += 1
+                        elif message['agent_sentiment'] == 'Negative':
+                            negative += 1
+                    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)